Put all symmetricizing function into class & documented it

jiroger opened the pull request:
Let me know if I should put more documentation. Also updated requirements file to include packages installed.

ritvikvasan commented on emm/bin/symmetricize.py:
```diff
@@ -0,0 +1,144 @@
+import scipy.io as sio
+import similaritymeasures
+
+class Symmetricize:
```
add an init method to the class

ritvikvasan commented on requirements_rtd.txt:
```diff
@@ -16,10 +15,12 @@ cycler==0.10.0
 decorator==4.4.1
 defusedxml==0.6.0
 docutils==0.15.2
+-e git+https://github.com/ritvikvasan/emm@01778567adcfef77f7bbf80747feb9e27caea9d9#egg=emm
```
should prob delete this

jiroger commented:
Alright, so I pushed in the changes. Chiefly among them, I totally rewrote the Symmetricize.py file to be more OOP-like and have an init method. I also split TestingNotebook into two: one for testing just Symmetricize.py and one for my development. Lastly, I removed the little extra blurb in requirements_rtd.txt
